spharpy/spherical.py

```diff
@@ -271,9 +282,21 @@ def spherical_harmonic_basis(n_max, coords):
     ----------
     n_max : integer
         Spherical harmonic order
-    coordinates : :doc:`pf.Coordinates <pyfar:classes/pyfar.coordinates>`
-        Coordinate object with sampling points for which the basis matrix is
+    coordinates : :doc:`pf.Coordinates <pyfar:classes/pyfar.coordinates>` or `sp.SamplingSphere <spharpy:classes/spharpy.samplings.coordinates>`
+        objects with sampling points for which the basis matrix is
         calculated
+    normalization : str, optional
+        Normalization convention, either 'n3d', 'maxN' or 'sn3d'.
+        The default is 'n3d'.
+        (maxN is only supported up to 3rd order)
+    channel_convention : str, optional
+        Channel ordering convention, either 'acn' or 'fuma'.
+        The default is 'acn'.
+        (FuMa is only supported up to 3rd order)
+    condon_shortley : bool or str, optional
+        Whether to include the Condon-Shortley phase term. If True or 'auto',
+        Condon-Shortley is included, if False it is not included.
+        The default is 'auto'.
```

Comment:
I understood this in a different way: In the basic functions we have this as a bool with the reasonable default, which is `True` for the complex valued and `False` (?) for the real valued bases.

In the SH class in the other pull, we will have `'auto'` as a default, which will pick `True` or `False` depending on the basis type (complex or real).

Does that make sense to you?

Reply:
Yes, generally makes sense to me, I just understood it the other way round. I don't have a strong opinion which method should get the 'auto' option. I am just wondering if it wouldn't be nice to also have it in the basis matrix generator method instead just the basis method. Let's discuss in the weekly.

Reply:
I don't mind both implementations. The current one would be a bit more consistent. 
(The current one is what we agreed on in the last meeting)

Reply:
We decided to leave everything as it is.